apps/services/handlers.py

```diff
@@ -201,28 +214,28 @@ async def read_hostel_accommodation(
         # (it's happened only if user request doesn't have review)
         return response
 
+    @staticmethod
     async def read_request_details(
-        self,
         *,
         request: Request,
         university_id: int,
         user_request_id: int,
         session: AsyncSession,
-    ):
+    ) -> UserRequestDetailsViewOut:
         return await user_request_detail_service.read(
             session=session,
             data={"university_id": university_id, "user_request_id": user_request_id},
         )
 
+    @staticmethod
     async def read_user_document(
-        self,
         *,
         request: Request,
         university_id: int,
         user_document_id: int,
         user: UserOut,
         session: AsyncSession,
-    ):
+    ) -> str:
```

Comment:
bytes, no?

Reply:
No, str. Method return filename with root. Bytes are returned by router function

Reply:
ok, got it